spring-kafka/src/test/java/org/springframework/kafka/support/serializer/RetryingDeserializerTests.java

```diff
@@ -54,6 +59,19 @@ void testRetry() {
 		rdes.close();
 	}
 
+	@SuppressWarnings("unchecked")
+	@Test
+	void retryingDeserializerWithRecoveryCallback() throws Exception {
+		RetryingDeserializer<String> rdes = new RetryingDeserializer<>((s, b) -> {
+			throw new RuntimeException();
+		}, new RetryTemplate());
+		RecoveryCallback<String> recoveryCallback = mock(RecoveryCallback.class);
```

Comment:
I know that is going to be nit-picking, but I'll shoot it anyway.

The Mockito starting with some version does not need the type argument.
Just `RecoveryCallback<String> recoveryCallback = mock();` is enough to make it working.
And the code looks nicer.